### DISCUSSION TITLE
how do i get those top languages cards on my profile

### DISCUSSION THREAD

**Discussion by donmegalo573** [community/community#19142] [Other Feature Feedback, Questions, & Ideas] [answered]
im new to github, couldn't find the answer on the forum

**Comment by justin-2028** [comment, marked as answer]
Take a look at this website and scroll down to the Top Languages Card section:
https://betterprogramming.pub/3-steps-to-improve-your-github-overview-page-950c64d4d465

Place the link on the article to your README file and the languages card should show on your profile.

hope that helps

**Reply by donmegalo573** [reply]
i'll figure out how to make a readme for my profile but thx

**Comment by byReqz** [comment]
mine are from here https://github.com/anuraghazra/github-readme-stats

**Comment by ZekeScript** [comment]
I use https://github.com/anuraghazra readme file
Just copy your favorite and read some documentation to setup.

Thats all!
